src/net.cpp

```diff
@@ -747,8 +747,9 @@ void V1TransportSerializer::prepareForTransport(CSerializedNetMsg& msg, std::vec
     CVectorWriter{SER_NETWORK, INIT_PROTO_VERSION, header, 0, hdr};
 }
 
-size_t CConnman::SocketSendData(CNode *pnode) const EXCLUSIVE_LOCKS_REQUIRED(pnode->cs_vSend)
+size_t CConnman::SocketSendData(CNode *pnode) const
```

Comment:
9d47d6da524b78ee4a628119f09636b855e881a2

nit, clang-format suggests:
```suggestion
size_t CConnman::SocketSendData(CNode* pnode) const
```